Tools/scripts/pathfix.py

```diff
@@ -164,12 +180,28 @@ def fix(filename):
     # Return success
     return 0
 
+
+def parse_shebang(shebangline):
+    end = len(shebangline)
+    start = shebangline.find(b' -', 0, end)  # .find() returns index at space
+    if start == -1:
+        return b'', b''
```

Comment:
I don't understand why it returns a tuple. Why not returning b''? It looks like a bug.